### DISCUSSION TITLE
I am not attending classes at the school that awards my degree, so I cannot pass the verification

### DISCUSSION THREAD

**Discussion by ttdxb** [community/community#65107] [GitHub Education]
### Hi everyone,

I'm a student from China. My university, A, has established an agreement with another institution, college B. While my degree is conferred by university A, I'm actually attending classes at college B. Additionally, my academic transcripts are under the management of college B, which is also authorized to issue Associate Degree certificates in China.

The current predicament I'm facing pertains to my student ID originating from university A. Because of my enrollment at college B, university A hasn't promptly updated my student ID. Consequently, I'm unable to substantiate my student status by uploading my student ID. My intention is to utilize a certification from the CHSI website, an official platform for educational verification in China, to validate my enrollment at the school. However, GitHub is indicating that I can't verify using this method. Notably, CHSI is the authoritative source for academic records verification in China.

Subsequently, I plan to authenticate my student status by submitting my academic transcript. The challenge here is that my email is affiliated with university A, while the transcript originates from college B. During the submission process, GitHub is also signaling that the verification cannot be completed. How should I navigate this situation to successfully accomplish my student authentication?

**Comment by ttdxb** [comment]
Adding a bit more information, as I'm from China, my name is in Chinese characters. However, due to GitHub's limitation in using Chinese usernames, I've resorted to using the romanized version of my name. Unfortunately, I can't utilize the romanized version of my name on the submitted educational verification since Chinese university websites typically don't provide English versions of enrollment certificates. I'm uncertain about the impact of this on my application. If it does indeed have an impact, what steps can I take to address this?

**Reply by arafajs** [reply]
@shuaidemeikan It's a complicated problem. I think these are the only things you can do to try to resolve this issue:

1. Contact the registrar's office at both universities A and B. Explain your situation to them and ask for their help in verifying your student status. They may be able to issue you a letter or certificate that GitHub will accept as proof of your enrollment.
2. Contact CHSI directly. They may be able to provide you with a more specific explanation of why GitHub is not accepting their verification. They may also be able to provide you with a different way to verify your student status that GitHub will accept.
3. Try to contact GitHub directly. Explain your situation to them and see if they can offer any suggestions.

**Reply by ttdxb** [reply]
thank you, These three methods are too troublesome for me, But still thank you for providing me with your feedback
